iina/MainMenuActions.swift

```diff
@@ -180,7 +180,7 @@ extension MainMenuActionHandler {
 
   @objc func menuChangeCrop(_ sender: NSMenuItem) {
     if let cropStr = sender.representedObject as? String {
-      player.setCrop(fromString: cropStr)
+      let _ = player.setCrop(fromString: cropStr)
```

Comment:
I believe you don't need the `let` here if you're just trying to discard the result.

Reply:
We should fix it in a future commit since `let _ = ...` also exists in other parts of our codebase.